resolwe_bio/tests/processes/test_pathway_map.py

```diff
@@ -0,0 +1,13 @@
+# pylint: disable=missing-docstring
+from resolwe.test import tag_process
+
+from resolwe_bio.utils.test import BioProcessTestCase
+
+
+class PathwayMapTestCase(BioProcessTestCase):
+
+    @tag_process('upload-metabolic-pathway')
+    def test_upload_metabolic_pathway(self):
+        inputs = {'src': 'matabolic-pathway.json.gz'}
```

Comment:
Rename the test file so that it includes the whitespace, e.g. `matabolic pathway.json.gz`. In the process, you might need to put quotes around the NAME variable: `re-save pathway "${NAME}.json"`